### DISCUSSION TITLE
Possible to have NumberOfTestSuites in XML-export from FB_xUnitXmlPublisher?

### DISCUSSION THREAD

**Discussion by TatMig** [tcunit/TcUnit#176] [Q&A] [answered]
Hi @sagatowski,
thank you for creating and providing a very useful library TcUnit!

A have a question about TcUnit.FB_xUnitXmlPublisher, method LogTestSuiteResults.
Here all tests results are written to the XML file, also  NumberOfSuccessfulTestCases and NumberOfFailedTestCases but not the NumberOfTestSuites. It is possible to add it to the XML file?

Thanks in advance for your help

**Comment by sagatowski** [comment, marked as answer]
@TatMig, thank you for your question!

I don't think it's possible as the XML-file has to be JUnit XML-compatible. In the [definition](https://llg.cubic.org/docs/junit/) ([XSD-schema](https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd)) there simply is no field for the number of test suites.

**Reply by Aliazzzz** [reply]
Good catch!

**Reply by TatMig** [reply]
Thank you for your prompt reply.

**Reply by sagatowski** [reply]
@TatMig If you find this framework useful then don't forget to star it, so other can discover it too! https://github.com/tcunit/tcunit/stargazers
